.eslintrc.js

```diff
@@ -8,11 +8,18 @@ module.exports = {
     'dist',
     'docs',
     'coverage',
+    'scripts',
+    'tests',
```

Comment:
Could we remove these entries? Locally the auto-fixer seems to handle them just fine.

We have a lot of code on `tests` directories that we want to stay linted. Nice to have the scripts linted as well.


```suggestion
```